packages/documentation-framework/layouts/sideNavLayout/sideNavLayout.js

```diff
@@ -94,20 +120,49 @@ const HeaderTools = ({
         <ToolbarGroup align={{ default: 'alignEnd' }}>
           {hasDarkThemeSwitcher && (
             <ToolbarItem>
-              <ToggleGroup aria-label="Dark theme toggle group">
-                <ToggleGroupItem
-                  aria-label="light theme toggle"
-                  icon={<SunIcon />}
-                  isSelected={!isDarkTheme}
-                  onChange={toggleDarkTheme}
-                />
-                <ToggleGroupItem
-                  aria-label="dark theme toggle"
-                  icon={<MoonIcon />}
-                  isSelected={isDarkTheme}
-                  onChange={toggleDarkTheme}
-                />
-              </ToggleGroup>
+              <Select
+                id="ws-theme-select"
+                isOpen={isThemeSelectOpen}
+                selected={themeMode}
+                onSelect={handleThemeChange}
+                onOpenChange={(isOpen) => setIsThemeSelectOpen(isOpen)}
+                toggle={(toggleRef) => (
+                  <MenuToggle
+                    ref={toggleRef}
+                    onClick={() => setIsThemeSelectOpen(!isThemeSelectOpen)}
+                    isExpanded={isThemeSelectOpen}
+                    icon={getThemeIcon(themeMode)}
+                    aria-label="Theme selection"
```

Comment:
This would apply to the example js file as well, but for the aria-label, instead of just this string it'd be more informative if we include the current theme selection so that the toggle is announced as "Dark theme, Theme selection".

A wai aria example of a combobox results in a similar accessible name structure (though it kinda gets it by default because their toggle is a combobox role, so it has a `value` attribute that gets included).

We'd need to do something like ```aria-label={`${getThemeDisplayText(themeMode)}, Theme selection`}``` (though creating a variable for the getThemeDisplayText would be better so that we can use that variable here and on line 137)